code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -297,7 +297,7 @@
 /mob/living/simple_animal/proc/adjustHealth(amount)
 	if(status_flags & GODMODE)
 		return 0
-	bruteloss = Clamp(bruteloss + amount, 0, maxHealth)
+	bruteloss = Clamp(bruteloss + (amount * config.damage_multiplier), 0, maxHealth)
```

Comment:
You should do this in the damage procs so that simple animals with non-inheriting adjustHealth() procs, like holoparasites, get increased damage.